src/Components/Genre/Genre.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import classes from "./Genre.module.css"
+
+class Genre extends React.PureComponent {
```

Comment:
The task was about creating Genre toggle and although the task is given poorly on details, it has statement that components should be interactive... here we can select one of the genres, but it doesn't effect anything. there is no state handling or reacting to user choices.

The same with Search component. it doesn't do anything.